puzzlejs/puzzle.css

```diff
@@ -161,6 +171,59 @@
     font-family: sans-serif;
 }
 
+.puzzle-entry .pos-top-left, .puzzle-entry .pos-top-right, .puzzle-entry .pos-bottom-left, .puzzle-entry .pos-bottom-right, .puzzle-entry .pos-top, .puzzle-entry .pos-bottom {
```

Comment:
Sorry, I don't understand what you're asking for.

Reply:
instead of a list of positions, a list of labels, e.g. `.link` (which you might rename to `.link-label`, and in a subsequent PR I will extend this positioning support to clues and extracts, so it'll eventually be `.link-label, .clue-label, .extract-label`)

Reply:
[I mean, you can do that extension if you want, but I'm trying to not scope-creep your work here :-)]

Reply:
I think I see... so have the shared stuff for `.link-label, .clue-label`, then followed by the individual `.pos-Y-X` classes with the unique settings? Should the default include something for the fields set by `.pos-*` classes, just in case one of them isn't set?

Reply:
> I think I see... so have the shared stuff for `.link-label, .clue-label`, then followed by the individual `.pos-Y-X` classes with the unique settings?
Yes.

> Should the default include something for the fields set by `.pos-*` classes, just in case one of them isn't set?
No need IMO.